.changes/v2.20.0/527-features.md

```diff
@@ -0,0 +1,3 @@
+* Added support for Defined Interfaces with client methods `VCDClient.CreateDefinedInterface`, `VCDClient.GetAllDefinedInterfaces`,
```

Comment:
Maybe something like this for extra clarity:

```suggestion
* Added support for Runtimed Defined Entity Interfaces with client methods `VCDClient.CreateDefinedInterface`, `VCDClient.GetAllDefinedInterfaces`,
```

I'm not sure though after looking at CD docs:
https://docs.vmware.com/en/VMware-Cloud-Director/10.4/VMware-Cloud-Director-Service-Provider-Admin-Portal-Guide/GUID-0749DEA0-08A2-4F32-BDD7-D16869578F96.html

As I see there:
- Runtime Defined Entity
- Defined Entity Interface
- RDE

So, what is the most specific "full" name for Defined Interfaces which we could use in the changelog and similar places?

Reply:
I think it would be Runtime Defined Entity Interface, although in API docs they're just referred as `Defined Interface`: 
https://developer.vmware.com/apis/vmware-cloud-director/v37.1/defined-interface/

Reply:
Yes, in the docs there's a section "Runtime Defined Entity Interfaces". So, let's have this full name in the change log for a better search ability.

Reply:
Changed 👍